Framework/PythonInterface/mantid/kernel/_aliases.py

```diff
@@ -26,6 +26,8 @@ def lazy_instance_access(cls):
 
 
     :param cls: The singleton class type
+    :param contains_as_str: When ``True``, ``__contains__`` will return ``False`` when looking for ``None``
+    and will convert the object to search for to a str
```

Comment:
Took me awhile to understand the logic here:

- Default (Original) behavior:  `contains_as_str` set to `False`, and everything else behaves exactly like before.
- New behavior: `contains_as_str` set to `True`, and now non-string type object can be checked with `mtd` using `in` statement (internally forced converted to str before passing back to `mtd`).